pkg/objstore/objtesting/acceptance_e2e_test.go

```diff
@@ -62,7 +63,10 @@ func TestObjStore_AcceptanceTest_e2e(t *testing.T) {
 			seen = append(seen, fn)
 			return nil
 		}))
-		testutil.Equals(t, []string{"obj_5.some", "id1/", "id2/"}, seen)
+		expected := []string{"obj_5.some", "id1/", "id2/"}
+		sort.Strings(expected)
+		sort.Strings(seen)
```

Comment:
This changes the interface invariants (: I don't think we rely on this though, so "should be" fine. Have you expierience SWIFT iterating in random order? Or just differently ordered?

Reply:
SWIFT iteration is deterministic, that is, it returns in the same order each time it is queried. It is just differently ordered, which causes the test to fail.